test/common.js

```diff
@@ -675,3 +675,8 @@ exports.getArrayBufferViews = function getArrayBufferViews(buf) {
   }
   return out;
 };
+
+// Crash the process on unhandled rejections.
+exports.crashOnUnhandledRejection = function() {
+  process.on('unhandledRejection', (err) => setImmediate(() => { throw err; }));
```

Comment:
You might want to do the crashing in a `nextTick` rather than a `setImmediate` so that further I/O does not get a chance to run.